diff --git a/console-client/Makefile b/console-client/Makefile
index 024a053d..097db89e 100644
--- a/console-client/Makefile
+++ b/console-client/Makefile
@@ -112,12 +112,12 @@ pc-kbd-CFLAGS = -DXKB_SUPPORT -DXKB_DATA_DIR=\"$(XKB_BASE)\" $(X11_CFLAGS)
 $(foreach XKB_UNIT, $(XKB_UNITS), $(eval $(XKB_UNIT)-CFLAGS = $(X11_CFLAGS)))
 compose-CFLAGS += -DDATADIR=\"$(datadir)\"
 pc_kbd-LDLIBS = $(X11_LIBS)
-install: $(XKB_BASE) $(addprefix $(XKB_BASE)/, $(XKB_DATA_FILES))
+install: $(XKB_BASE) $(prefix)$(addprefix $(XKB_BASE)/, $(XKB_DATA_FILES))
 
 $(XKB_BASE):
 	@$(MKINSTALLDIRS) $@
 
-$(XKB_BASE)/%: xkb/xkb-data/%
+$(prefix)$(XKB_BASE)/%: xkb/xkb-data/%
 	$(INSTALL_DATA) $< $@
 
 kstoucs.o: xkb/kstoucs_map.c
